### PR TITLE
conda: Remove ambiguous packages

### PR DESCRIPTION
**Pull request by HaoZeke**
Hi.

I'm not really very familiar with `conda`, but I think this seems like a minimal set of packages which are present on all major operating systems.

Rohit

----
This PR is part of the **Google Season of Docs 2020**

### CODE REVIEW

**Comment by isuruf** [comment]
You are thinking about it this is the wrong way. Don't try to go from the full list to a minimal list. Forget about the existing list.

What exactly do you need to build the docs? Then list them.

**Comment by HaoZeke** [comment]
Excellent, now I've listed only the `conda install` commands I've used from my shell history

**Review by isuruf — APPROVED** [review]
Thanks

File: symedocs.yml [hunk 1]
```diff
@@ -3,250 +3,34 @@ channels:
...
+  - ipykernel
+  - ipywidgets
+  - jupyter
+  - jupyter_nbextensions_configurator
+  - jupyterlab
+  - jupytext
+  - matplotlib
+  - myst-nb
+  - numpy
+  - pandas
```
Comment (isuruf) [inline symedocs.yml]: pandas?

Comment (HaoZeke) [reply symedocs.yml]: Was planning to use it in a tutorial for `symengine.py`. I'll remove it for now.

**Pull request merged**